fiftyone/operators/operations.py

```diff
@@ -356,6 +356,8 @@ def register_panel(
                 current extended selection changes
             on_change_group_slice (None): an operator to invoke when the group
                 slice changes
+            on_change_spaces (None): an operator to invoke when the current spaces changes
```

Comment:
Is `on_change_workspace` needed? Comparing with view-related events for context, we only have `on_change_view`, not `on_change_view_name`.

`on_change_spaces` should fire every time that `on_change_workspace` fires, plus it would also fire when changing between two different unsaved spaces configurations.

Reply:
Agreed. Removing `on_change_workspace` as it's little redundant